build/kobo.mk

```diff
@@ -81,6 +81,7 @@ THIRDPARTY_TOOL_NAMES = simple_usbmodeswitch
 THIRDPARTY_TOOL_FILES = $(addprefix $(THIRDPARTY_LIBS_ROOT)/bin/,$(THIRDPARTY_TOOL_NAMES))
 
 KOBO_KERNEL_DIR = /opt/kobo/kernel
+CA_FILE != ls -v $(DOWNLOAD_DIR)/cacert*.pem | tail -n  1
```

Comment:
This looks... fragile.
For example, it breaks when the file does not already exist.

Reply:
removed